integration/flink/src/main/java/io/openlineage/flink/utils/PathUtils.java

```diff
@@ -30,11 +31,11 @@ public static DatasetIdentifier fromURI(URI location, String defaultScheme) {
   private static String trimSlashesInName(String name) {
```

Comment:
It looks like the method removes the first and the last slash in the string. 
This can be achieved with `StringUtils.removeFirst` and `StringUtils.removeEnd`:
https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#removeEnd-java.lang.String-java.lang.String-

Reply:
Fixed in previous PR